tests/search/test-class-search.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Automattic\VIP\Search;
 
+require_once __DIR__ . '/../../search/search.php';
```

Comment:
I moved this up so I have access to the Search constants in the data providers (that seems to run before setup).